widget/class-widget.php

```diff
@@ -61,6 +64,12 @@ public function widget( $args, $instance ) {
             )
          );
 
+         // Set default title if empty
+         $instance['title'] = empty( $instance['title'] ) ? 'Flexible Volunteer Opportunities' : $instance['title'];
```

Comment:
I'm really on the fence with the defaults. Right now if you leave the title blank you would have no indication that it's going to say "Flexible Volunteer Opportunities" or "One-Time Volunteer Opportunities" in the title. What do you think? I'm back and forth on this one.